crates/holochain/src/sweettest/sweet_conductor.rs

```diff
@@ -22,6 +23,9 @@ pub type SignalStream = Box<dyn tokio_stream::Stream<Item = Signal> + Send + Syn
 /// A useful Conductor abstraction for testing, allowing startup and shutdown as well
 /// as easy installation of apps across multiple Conductors and Agents.
 ///
+/// SweetConductors are meant to run on a single machine. Peer discovery is
+/// explicit (TODO), and they are not expected to use a bootstrap server.
+///
```

Comment:
We already have tests that do explicit peer discovery though?

Reply:
Oh yeah I should clarify that I want to be able to discover peers in a more fine grained way than "everyone discovers everyone else"

Reply:
BTW this is done now in the next PR